test/integration_test.go

```diff
@@ -38,14 +38,13 @@ import (
 	"testing"
 	"time"
 
-	"go.opentelemetry.io/otel/baggage"
-
 	"github.com/opentracing/opentracing-go"
 	"github.com/pborman/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 	"github.com/uber-go/tally/v4"
+	"go.opentelemetry.io/otel/baggage"
```

Comment:
This was moved by my IDE and I was about to move it back, but it seems better now.